cmake/Modules/installviapip.cmake

```diff
@@ -9,9 +9,9 @@ execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip freeze OUTPUT_VARIABLE PYTHO
         execute_process(COMMAND pip freeze OUTPUT_VARIABLE PYTHON_PACKAGE_LIST)
         if ("${PYTHON_PACKAGE_LIST}" STREQUAL "")
             message(WARNING "Failed to find pip. Pip is required to automatically install ${module}. Will now install pip.")
-            file(DOWNLOAD https://bootstrap.pypa.io/get-pip.py ./get-pip.py --user STATUS SUCCESS)
```

Comment:
Any issue with installing pip to a user folder instead of the global directories?

Reply:
This is just the download line, this option does not belong there.